src/app/parcels/ActionBar/DuplicateDownloadWarning.tsx

```diff
@@ -0,0 +1,28 @@
+"use client";
+
+import React from "react";
+import Alert from "@mui/material/Alert/Alert";
+import styled from "styled-components";
+import { displayPostcodeForHomelessClient } from "@/common/format";
+
+export interface DuplicateDownloadWarningProps {
+    postcodes: (string | null)[];
+}
+
+const StyledAlert = styled(Alert)`
+    border-radius: 4px;
+`;
+
+const DuplicateDownloadWarning: React.FC<DuplicateDownloadWarningProps> = (props) => {
+    return (
+        <StyledAlert severity="warning">
+            The following postcodes have already been downloaded:{" "}
```

Comment:
I was in a fight with the linter and lost :( it added this so that the whitespace didn't get ignored because it was at the end of the line. I've reorganised that code now to make it clearer.